src/execution/sql/functions/string_functions.cpp

```diff
@@ -363,4 +363,10 @@ void StringFunctions::Right(UNUSED_ATTRIBUTE exec::ExecutionContext *ctx, String
   }
 }
 
+// TODO(Deepayan): Do we need to print out OS name/version/compiler?
+void StringFunctions::Version(UNUSED_ATTRIBUTE exec::ExecutionContext *ctx, StringVal *result) {
+  const char *version = "NoisePage 1.0.0";
```

Comment:
We should `#define` these somewhere other than in this function. I think the network protocol needs it too.